autoload/lightline/bufferline.vim

```diff
@@ -9,6 +9,7 @@ let g:loaded_lightline_bufferline = 1
 
 let s:filename_modifier = get(g:, 'lightline#bufferline#filename_modifier', ':.')
 let s:modified          = get(g:, 'lightline#bufferline#modified', '+')
+let s:read_only         = get(g:, 'lightline#bufferline#read_only', '')
```

Comment:
We should use `-` as default read-only indicator because the unicode symbol is not available in all fonts. If someone wants to use the unicode symbol it can be explicitly set via this option.